source/EventDetail.html.md.erb

```diff
@@ -7,8 +7,7 @@ tags: []
 Event Details and Resources
 ------
 
-Downloadable <%= data.event.year %> Docs (links are not likely to work until
```

Comment:
<%= data.event.year %> is automatically populated  from a central file. You wouldn't change it here instead it's changed once and it is updated everywhere.

Reply:
Understood!